homeassistant/components/supervisord/sensor.py

```diff
@@ -71,7 +71,9 @@ def device_state_attributes(self):
     def update(self):
         """Update device state."""
         try:
-            self._info = self._server.supervisor.getProcessInfo(self._info.get("name"))
+            self._info = self._server.supervisor.getProcessInfo(
+                self._info.get("group") + ":" + self._info.get("name")
```

Comment:
If I recall correctly, a supervisord process doesn't have to be in a group.
That would cause it to become `:name`?

Reply:
if a supervisord process doesn't in a group, The group is the same as the name
like "name:name"